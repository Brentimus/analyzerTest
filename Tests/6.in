1...2.2...2.2..12.3..1.2.3.12.3..3.3.2