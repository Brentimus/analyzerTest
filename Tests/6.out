1  1  Integer  1  1
1  2  Separator  Doubledot  ..
1  4  Separator  Dot  .
1  5  Double  2.2000000000000002E+000  2.2
1  8  Separator  Doubledot  ..
1  10  Separator  Dot  .
1  11  Double  2.2000000000000002E+000  2.2
1  14  Separator  Doubledot  ..
1  16  Double  1.2300000000000001E+001  12.3
1  20  Separator  Doubledot  ..
1  22  Double  1.2000000000000000E+000  1.2
1  25  Separator  Dot  .
1  26  Double  3.1200000000000001E+000  3.12
1  30  Separator  Dot  .
1  31  Integer  3  3
1  32  Separator  Doubledot  ..
1  34  Double  3.2999999999999998E+000  3.3
1  37  Separator  Dot  .
1  38  Integer  2  2
1  39  Eof